packages/@n8n/nodes-langchain/nodes/agents/Agent/agents/ToolsAgent/execute.ts

```diff
@@ -93,7 +93,43 @@ export async function toolsAgentExecute(this: IExecuteFunctions): Promise<INodeE
 	const tools = (await getConnectedTools(this, true, false)) as Array<DynamicStructuredTool | Tool>;
 	const outputParser = (await getOptionalOutputParsers(this))?.[0];
 	let structuredOutputParserTool: DynamicStructuredTool | undefined;
+	/**
+	 * Handles the agent text output and transforms it in case of multi-output.
```

Comment:
Can you please add some description why do we need this? Or examples of cases when this will be used.

Reply:
Added